### PR TITLE
Fix warning design test 4

### PR DESCRIPTION
**Pull request by JorisMonnet**

### CODE REVIEW

**Review by Marty99-dev — APPROVED** [review]
We must not  forget to remove unused resources !

File: app/src/main/java/ch/epfl/sdp/blindwar/data/SpotifyApiConstants.kt [hunk 1]
```diff
@@ -4,8 +4,8 @@ import android.util.Base64
 
 object SpotifyApiConstants {
     /** Credentials constants **/
-    const val CLIENT_ID = "66f4e3c14d6e425caed973ba62bb2077";
-    const val CLIENT_SECRET = "8b654e6a74a54dd5a6300bfadecc3c8a";
+    const val CLIENT_ID = "66f4e3c14d6e425caed973ba62bb2077"
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/data/SpotifyApiConstants.kt:7]: Shouldn't it be removed ?

**Review by marluxiaboss — APPROVED** [review]
I notice a lot of commented code. It makes the code overall more messy than necessary.

File: app/src/main/java/ch/epfl/sdp/blindwar/data/SpotifyApiConstants.kt (hunk 1 above)
Comment (JorisMonnet) [reply app/src/main/java/ch/epfl/sdp/blindwar/data/SpotifyApiConstants.kt:7]: > I notice a lot of commented code. It makes the code overall more messy than necessary.

Yes, that's true

> Shouldn't it be removed ?

@ghemDD  is working on it, secret are modified every time so don't worry with these values

**Pull request merged**